README.md

```diff
@@ -28,10 +28,10 @@ Our goal is faster iteration on massive tabular datasets, both for experimentati
 ### Getting Started
 NVTabular is available in the NVIDIA container repository at the following location: http://ngc.nvidia.com/catalog/containers/nvidia:nvtabular.
 
-Currently we have the beta release (0.2) container, you can pull the container using the following command:
+Currently we have the beta release (0.3) container, you can pull the container using the following command:
```

Comment:
We don't have a container up at this link yet - lets remove this change for now, and make this change when we have published the new NGC container